specs/subresourceintegrity/spec.markdown

```diff
@@ -127,11 +127,11 @@ The term <dfn>origin</dfn> is defined in the Origin specification.
 [[!RFC6454]]
 
 The terms <dfn>secure document</dfn> and
-<dfn>secure context</dfn> are defined in [section 2 of the Secure
+<dfn>secure context</dfn> are defined in section 2 of the [Secure
 Contexts][securecontext] specification. An example of a secure document is a
 document loaded over HTTPS. A counterexample is a document loaded over HTTP.
 
-[securecontext]: https://w3c.github.io/webappsec/specs/powerfulfeatures/#terms
+[securecontext]: http://www.w3.org/TR/powerful-features/
```

Comment:
It has changed to `#framework` in the latest editor's draft but that's currently not in the last published draft. Given it's not yet fully stable, I figured we're probably best just linking to the spec itself.